cmd/netsim/main.go

```diff
@@ -61,17 +61,18 @@ func main() {
 		log.Crit("main", "error", err)
 	}
 
-	sim, err := p2p.NewSimulator(num)
+	sim, err := NewSimulator(num)
 	if err != nil {
 		log.Crit("NewSimulator", "error", err)
 	}
 
-	defer func() {
-		err = sim.IpfsNode.Close()
-		if err != nil {
-			log.Warn("main", "warn", err.Error())
-		}
-	}()
+	// TODO: Since we removed IPFS this needs to handle error channel from sim.Bootnode.Stop()
```

Comment:
do you wanna add this? or do it another time?

Reply:
The whole simulator needs love, so I think its fine to leave for now

Reply:
sure